src/app/admin/createCollectionCentre/CreateCollectionCentreForm.tsx

```diff
@@ -7,26 +7,37 @@ import {
     checkErrorOnSubmit,
     Errors,
     FormErrors,
-    setError,
-    setField,
+    createSetter,
+    CardProps,
 } from "@/components/Form/formFunctions";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faBuildingCircleArrowRight } from "@fortawesome/free-solid-svg-icons";
 import RefreshPageButton from "@/app/admin/common/RefreshPageButton";
-import { InsertSchema } from "@/databaseUtils";
-import NameCard from "@/app/admin/createCollectionCentre/NameCard";
 import AcronymCard from "@/app/admin/createCollectionCentre/AcronymCard";
 import supabase from "@/supabaseClient";
 import { logErrorReturnLogId, logInfoReturnLogId } from "@/logger/logger";
 import { AuditLog, sendAuditLog } from "@/server/auditLog";
 import Alert from "@mui/material/Alert/Alert";
+import NameCard from "./NameCard";
 
-const initialFields: InsertSchema["collection_centres"] = {
+interface CollectionCentreFields {
+    name: string;
+    acronym: string;
+}
+
+interface CollectionCentreErrors extends FormErrors<CollectionCentreFields> {
+    name: Errors;
+    acronym: Errors;
+}
```

Comment:
yes

Reply:
nevermind, I get an error saying that Required<FormErrors<SpecificFields>> is not assignable to FormErrors<SpecificFields> because the values in the latter can be undefined but in the former they have to be of type Errors. Not sure how to fix :(

Reply:
hm what was the problem here?

Reply:
I fixed it, struggling to remember details - sorry :(